pandas/io/stata.py

```diff
@@ -363,16 +326,16 @@ def convert_delta_safe(
         conv_dates = convert_year_month_safe(year, month)
     elif fmt.startswith(("%tq", "tq")):  # Delta quarters relative to base
         year = stata_epoch.year + dates // 4
-        month = (dates % 4) * 3 + 1
-        conv_dates = convert_year_month_safe(year, month)
+        quarter_month = (dates % 4) * 3 + 1  # type: Series
```

Comment:
This is an unwanted pattern. use py3.6 variable annotation syntax

Reply:
Thanks, I moved it.